plugins/notifications/slack/install.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Remove this file, as this plugin will be distributed with crowdsec, we don't need a dedicated file for the install.

Reply:
fixed via https://github.com/crowdsecurity/crowdsec/pull/857/commits/80e040abb7f9c2ca6661bf5568742dd0f09076f1